cynetdiff/models.pyx

```diff
@@ -1,3 +1,7 @@
+# cython: language_level=3str
+# distutils: language=c++
+# distutils: define_macros=NPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
```

Comment:
Specifying these manually isn't the best practice, these can be specified using meson. See how we set it up for SciPy [here](https://github.com/scipy/scipy/blob/5aa61cb0f4c96b83ded51b0e745dee8dfec3d613/scipy/meson.build#L380-L394)